app/models/application.py

```diff
@@ -44,25 +40,47 @@ def from_dict(cls, d: dict):
             }
         )
 
-    def create_sections(self, application):
+    @staticmethod
+    def create_blank_sections(fund_id, round_id, language):
```

Comment:
Could we add a test for this function?

Reply:
we have now changed this to a normal instance function and the Class is already being tested so this is being covered 👍🏾 :)